libs/logic-apps-shared/src/utils/src/lib/helpers/functions.ts

```diff
@@ -358,7 +358,7 @@ export function safeSetObjectPropertyValue(
  * @arg {any} target The target location to be merged at.
  * @arg {any} The source location to be merged from.
  */
-function deepMerge(target: any, source: any) {
+export function deepMerge(target: any, source: any) {
```

Comment:
@ccastrotrejo I'm not an expert in typescript, but doesn't this change expose this method as public API (that we would then need to add a bunch more testing for)?

Reply:
@mireedmsft thats right, we will need to make unit test for the `deepMerge` function. You can add tests over there or I found out there is another function (which already has unit test and its exported) that does the same thing as the `deepMerge`.  @Jaden-Codes You can use the `extend` function. You can use it as this 
`copiedValue = extend({}, complexObject, { key3: 1 }),`

Reply:
Resolved this in the current pr.